### DISCUSSION TITLE
[244850] Space Engineers | 本体 | 清单更新

### DISCUSSION THREAD

**Discussion by Nobody2613** [pjy612/SteamManifestCache#88] [Share]
[244850] Space Engineers | 本体
```
appid:244850
keys:
1241550;0f4349457367949e9aca25e33826ae022cd451a67b5ad5761a719d7838caa2f9
244851;c2a914f1b4a29930055cf542e9061ead9b68a11f83da2bb811ff0bdd6ccd3fc4
573160;90b858382780b17809d769a3c519d42315dec52ac14f62cc36a730028c8c2d55
```
[244850_manifests.zip](https://github.com/pjy612/SteamManifestCache/files/13703697/244850_manifests.zip)

**Comment by pjy612** [comment]
好像仓库内是新的，不过缺了个 [573900](https://steamdb.info/depot/573900/)

**Reply by Nobody2613** [reply]
DLC [573900](https://steamdb.info/depot/573900/) 文件内容包含在 DLC [573160](https://steamdb.info/depot/573160/)
又看了一下仓库，好像都是最新的清单
无需更新，打扰了

**Reply by pjy612** [reply]
也就是 573900 不重要了 有 [573160](https://steamdb.info/depot/573160/) 就行是吧。
了解了。。。（没玩过这个

**Reply by Nobody2613** [reply]
对，主要是已经没有途径获取573900了
